packages/dd-trace/test/appsec/rasp.express.plugin.spec.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const axios = require('axios')
+const agent = require('../plugins/agent')
+const getPort = require('get-port')
+const appsec = require('../../src/appsec')
+const Config = require('../../src/config')
+const path = require('path')
+const { assert } = require('chai')
+
+withVersions('express', 'express', expressVersion => {
+  describe('RASP', () => {
+    let app, server, port
+
+    before(() => {
+      return agent.load(['http'], { client: false })
+    })
+
+    before((done) => {
+      const express = require(`../../../../versions/express@${expressVersion}`).get()
+      const expressApp = express()
+
+      expressApp.get('/', (req, res) => {
+        app(req, res)
+      })
+
+      getPort().then(newPort => {
+        port = newPort
+        server = expressApp.listen(port, () => {
+          done()
+        })
+      })
+    })
+
+    beforeEach(() => {
+      appsec.enable(new Config({
```

Comment:
do we really need to disable/enable before and after each ? or could we just put it in `before()` ?